serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/MatrixClient.kt

```diff
@@ -84,6 +85,79 @@ class MatrixSession(val client: HttpClient, val access_token: String, var transa
         }
     }
 
+    fun sendImageMessage(url: String, room_id: String): Outcome<String> {
+        try {
+            val result = runBlocking {
+                val img_f = File(url)
+                val image_data = img_f.readBytes()
+                val f_size = image_data.size
+                var ct = ContentType.Image.JPEG
+                val mimetype =
```

Comment:
this is a tad odd - maybe val (ct, mimetype) = if (...) { (ContentType.Image.PNG, "image/png") } else ...?

Reply:
That is cleaner, to be honest I just wasn't sure of the syntax of the paired return and I remembered to set the content type and mimetype at different times when originally writing. @enoxone also mentioned this to me. As before, I can also change this in the audio branch